### ISSUE TITLE
Equation 9

### ISSUE THREAD

**Issue by marzi-heidari** [YDU-uva/ProtoDiff#2]
I'm confused about equation 9 in the paper. As far as I know, the output for the diffusion model should be the predicted noise but the loss in equation 9 is computed as the output supposed to be the denoised overfitted prototype.

Another question about this equation is how you encounter $\tilde{z}$ as an additional input as, again, as far as I recall defusion models usually take as input the noisy input and the time-step. 

It would be great if you shared the code snippet for this equation and the diffusion model.

**Comment by YDU-uva** [comment]
Hi Marzi:

We have mentioned it in the section meta-training phase, “However, we discovered empirically that signal prediction performs better than noise prediction in our experiments, so we parameterized the diffusion model to output the prototype.”

Currently, I can share the initial code for you, pls email me. You can found my email in my personal website.

Best,
Yingjun

2023年10月30日 15:53，Marzi Heidari ***@***.***> 写道：



I'm confused about equation 9 in the paper. As far as I know, the output for the diffusion model should be the predicted noise but the loss in equation 9 is computed as the output supposed to be the denoised overfitted prototype.

Another question about this equation is how you encounter $\tilde{z}$ as an additional input as, again, as far as I recall defusion models usually take as input the noisy input and the time-step.

It would be great if you shared the code snippet for this equation and the diffusion model.
